packages/components/search/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "search",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "axios": "^0.24.0",
+    "core-js": "^3.6.5",
+    "vue": "^2.6.11"
+  },
+  "devDependencies": {
+    "@action-agenda/cached-apis": "~1.0.7",
+    "@action-agenda/icons": "~1.0.1",
+    "@houlagins/default-options": "~1.0.0",
```

Comment:
we want to remove this, and change code so it's not needed.

Reply:
cached-apis and icons are removed. default-options not removed. default-options is used in api.